packages/notebook/test/model.spec.ts

```diff
@@ -67,7 +67,7 @@ describe('@jupyterlab/notebook', () => {
         const cell = model.contentFactory.createCodeCell({});
         model.cells.push(cell);
         model.fromJSON(utils.DEFAULT_CONTENT);
-        expect(ArrayExt.firstIndexOf(toArray(model.cells), cell)).toBe(-1);
+        expect(ArrayExt.firstIndexOf(Array.from(model.cells), cell)).toBe(-1);
```

Comment:
Could we replace it by `findIndex`?

```suggestion
        expect(findIndex(model.cells, c => c === cell)).toBe(-1);
```

https://github.com/jupyterlab/lumino/blob/0e652ad10d8f0e90d96aca781cc691facd28a50b/packages/algorithm/src/find.ts#L88